cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -83,7 +83,7 @@ public static void deleteGCSPath(Storage storageClient, String bucket, String pa
       boolean addedToDelete = false;
       for (Blob blob : blobs.iterateAll()) {
         LOG.trace("Added path to be deleted {}", blob.getName());
-        batch.delete(blob.getBlobId(), Storage.BlobSourceOption.generationMatch());
```

Comment:
Seems like you brought in changes from develop branch? They shouldn't be here.

Reply:
Oh no, this commit gives an error with the way I am updating modules. So reverted it for testing. It is not part of the final PR